prac_07/guitar.py

```diff
@@ -33,10 +33,14 @@ def is_vintage(self):
     def __lt__(self, other):
         return self.year < other.year
 
+    def __iter__(self):
+        return iter([self.name, self.year, self.cost])
```

Comment:
I haven't seen __iter__ before
Any chance you could tell me how it works? I'm curious :)